README.md

```diff
@@ -1,2 +1,46 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+## The aim of this pseudocode is to show the process of selecting a secret word from a string of other words. Which has been selected by another user. 
+
+ 
+
+attemptNo � 4 
+
+wordLength � Number of letter in the word 
+
+secretWord � The secret word input 
+
+success � False 
+
+ 
+
+print (�Please guess the�, wordLength, �letter word.�) 
+
+ 
+
+while the attempt value is > 0 AND success = False 
+
+ 
+
+Likeness <- 0 
+
+guess <- get the user Input 
+
+if secretWord is eqaul to guess [i] 
```

Comment:
Presumably `i` is a loop counter here, however this is not clear from the pseudocode. Try something more like

```
for i = 0, ..., wordLength-1
   if secretWord[i] == guess[i]
      Likeness += 1
   end if
end for
```

to make the looping clear